walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,7 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+
+    return "Hello " + name.trim();
```

Comment:
FYI, it can be simpler to use String.format if there are multiple arguments.

ex
return String.format("Hello %s", name.trim());

But concatenation works too.

Reply:
Got it!